src/components/LoanCards/SharedLoanCardController.vue

```diff
@@ -0,0 +1,307 @@
+<template>
+	<div>
+		<kv-classic-loan-card
+			:loan-id="loanId"
+			:loan="loan"
+			:custom-loan-details="customLoanDetails"
+			:use-full-width="useFullWidth"
+			:show-tags="showTags"
+			:category-page-name="categoryPageName"
+			:enable-five-dollars-notes="enableFiveDollarsNotes"
+			:large-card="largeCard"
+			:is-adding="isAdding"
+			:is-visitor="isVisitor"
+			:basket-items="basketItems"
+			:is-bookmarked="isBookmarked"
+			:kv-track-function="$kvTrackEvent"
+			:photo-path="PHOTO_PATH"
+			:show-view-loan="showViewLoan"
+			:external-links="true"
+			:route="currentRoute"
+			:user-balance="userBalance"
+			:get-cookie="getCookie"
+			:set-cookie="setCookie"
+			@toggle-bookmark="toggleBookmark"
+			@add-to-basket="addToBasket"
+		/>
+	</div>
+</template>
+
+<script>
+import * as Sentry from '@sentry/vue';
+import { gql } from '@apollo/client';
+import { setLendAmount, handleInvalidBasket, hasBasketExpired } from '@/util/basketUtils';
+import { readLoanFragment, watchLoanData } from '@/util/loanUtils';
+import bookmarkLoan from '@/util/bookmarkUtil';
+import logFormatter from '@/util/logFormatter';
+import { createIntersectionObserver } from '@/util/observerUtils';
+import percentRaisedMixin from '@/plugins/loan/percent-raised-mixin';
+import timeLeftMixin from '@/plugins/loan/time-left-mixin';
+import loanCardFieldsExtendedFragment from '@/graphql/fragments/loanCardFieldsExtended.graphql';
+import KvClassicLoanCard from '~/@kiva/kv-components/vue/KvClassicLoanCard';
+
+const PHOTO_PATH = 'https://www-kiva-org.freetls.fastly.net/img/';
+
+const loanQuery = gql`
+	${loanCardFieldsExtendedFragment}
+	query kcBasicLoanCard($basketId: String, $loanId: Int!) {
+	shop (basketId: $basketId) {
+		id
+		basket {
+			id
+			# for isInBasket
+			items {
+				values {
+					id
+				}
+			}
+		}
+	}
+	lend {
+		loan(id: $loanId) {
+			id
+			...loanCardFieldsExtended
+		}
+	}
+	my {
+		id
+		userAccount {
+			id
+		}
+	}
+}`;
+
+export default {
+	name: 'SharedLoanCardController',
```

Comment:
Rename `KvClassicLoanCardContainer` to match CPS.

Reply:
I think the linter may force this to match the name of the file...

Reply:
Right, the file should be renamed too.